uber/templates/regform.html

```diff
@@ -357,7 +383,7 @@
 </div>
 
 <div class="form-group">
-    <label for="cellphone" class="col-sm-2 control-label">Your Cellphone Number</label>
+    <label for="cellphone" class="col-sm-2 control-label"><b>Your Cellphone Number</b></label>
     <div class="col-sm-6">
```

Comment:
You're probably right on that. I saw it was the only un-bolded field so I figured it was just overlooked or something. I'll fix that for the next commit.